ui/src/main/java/org/pentaho/di/ui/repository/kdr/KettleDatabaseRepositoryDialog.java

```diff
@@ -435,20 +435,6 @@ private void create() {
     getInfo( repositoryMeta );
 
     if ( repositoryMeta.getConnection() != null ) {
-      if ( repositoryMeta.getConnection().getAccessType() == DatabaseMeta.TYPE_ACCESS_ODBC ) {
-        // Show a warning: using ODBC is not always the best choice ;-)
-        System.out.println( "Show ODBC warning..." );
-
-        MessageBox qmb = new MessageBox( shell, SWT.ICON_WARNING | SWT.YES | SWT.NO );
-        qmb.setMessage( BaseMessages.getString( PKG, "RepositoryDialog.Dialog.ODBCIsNotSafe.Message", Const.CR,
```

Comment:
Please remove the "RepositoryDialog.Dialog.ODBCIsNotSafe.*" entries from the message files here: [ui/src/main/resources/org/pentaho/di/ui/repository/dialog/messages](https://github.com/pentaho/pentaho-kettle/tree/master/ui/src/main/resources/org/pentaho/di/ui/repository/dialog/messages)